submission/submission-webapp/src/main/resources/application-local.yml

```diff
@@ -36,3 +41,6 @@ app:
     magicDirPath: # Absolute path to the folder to be used for the user/groups magic folders links
     environment: LOCAL
     requireActivation: false
+  persistence:
+    enableMongo: false
+    connection: mongodb://root:admin@localhost:27017
```

Comment:
not following why we need this updates

Reply:
The purpose of this file is to have a ready to go application.yml file for local environments. We already have the [connection string for MySQL there](https://github.com/EBIBioStudies/biostudies-backend-services/pull/331/files/396f85a421cb49b5b0d2219b3b6437d8f5839407#diff-3fdf6382a2e250b6405e23bc785966755aaca13cbe545636bd0d56da307704eaL3), I'm just adding the missing mongo part.

Reply:
but here why we need this one, we use spring config so AFAIK this is not needed

Reply:
Dunno, you added this property in the application.yml. I'm just adding the local value here

Reply:
UPDATE: Apparently, this property isn't necessary so I'll just remove it